controllers/provisioning_controller.go

```diff
@@ -630,14 +618,18 @@ func (r *ProvisioningReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	klog.InfoS("Network stack calculation", "NetworkStack", r.NetworkStack)
 
+	provisioningFilter := predicate.NewPredicateFuncs(func(object client.Object) bool { return object.GetName() == metal3iov1alpha1.ProvisioningSingletonName })
```

Comment:
Would it be appropriate to log here if the name does not match?

Reply:
I do not see a need. The operator is interested in this particular resource, otherwise it just doesn't care.

Reply:
I'm worried about users creating a wrong resource and wondering why it's not reconciled.